### PR TITLE
kubespy 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by ddunkin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by commitay** [comment]
```diff
diff --git a/Formula/kubespy.rb b/Formula/kubespy.rb
index 79b756854a..46fe201f49 100644
--- a/Formula/kubespy.rb
+++ b/Formula/kubespy.rb
@@ -1,5 +1,5 @@
 class Kubespy < Formula
-  desc "Tools for observing Kubernetes resources in realtime, powered by Pulumi"
+  desc "Tools for observing Kubernetes resources in realtime"
   homepage "https://pulumi.io/"
   url "https://github.com/pulumi/kubespy.git",
       :tag => "v0.4.0",
@@ -8,14 +8,13 @@ class Kubespy < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "on"
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
     dir = buildpath/"src/github.com/pulumi/kubespy"
     dir.install buildpath.children
 
     cd dir do
       system "make", "build"
-      bin.install "#{dir}/kubespy"
+      bin.install "kubespy"
       prefix.install_metafiles
     end
   end
```

File: Formula/kubespy.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    cd dir do
+      system "make", "build"
+      bin.install "#{dir}/kubespy"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/kubespy", "version"
```
Comment (commitay) [inline Formula/kubespy.rb]: The test needs to exercise and verify the functionality of the software.

File: Formula/kubespy.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Kubespy < Formula
+  desc "Tools for observing Kubernetes resources in realtime, powered by Pulumi"
+  homepage "https://pulumi.io/"
```
Comment (commitay) [inline Formula/kubespy.rb]: Is there a specific homepage for this tool? I can't find any mention of `kubespy` at this domain.

Comment (ddunkin) [reply Formula/kubespy.rb]: No, there's just the GitHub repo and a blog post https://blog.pulumi.com/kubespy-and-the-lifecycle-of-a-kubernetes-pod-in-four-images. Maybe https://github.com/pulumi/kubespy would be better?

**Comment by ddunkin** [comment]
@commitay, anything else I need to change?

File: Formula/kubespy.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Kubespy < Formula
+  desc "Tools for observing Kubernetes resources in realtime"
+  homepage "https://github.com/pulumi/kubespy"
+  url "https://github.com/pulumi/kubespy.git",
+      :tag => "v0.4.0",
```
Comment (lblackstone) [inline Formula/kubespy.rb]:
```suggestion
      :tag      => "v0.4.0",
```

Comment (ddunkin) [reply Formula/kubespy.rb]: Great, thanks.

**Comment by lblackstone** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Thanks @ddunkin !

**Pull request closed without merging**